packages/Discretization/src/DTK_DiscretizationHelpers.hpp

```diff
@@ -0,0 +1,348 @@
+/****************************************************************************
+ * Copyright (c) 2012-2018 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef DTK_DISCRETIZATION_HELPERS
+#define DTK_DISCRETIZATION_HELPERS
+
+#include <DTK_Topology.hpp>
+
+#include <Kokkos_Macros.hpp>
+#include <Kokkos_View.hpp>
+
+namespace DataTransferKit
+{
+namespace Discretization
+{
+namespace Helpers
+{
```

Comment:
Do we declare nested namespaces anywhere else in DTK?  I don't necessarily have a problem with it, just surprised.

Reply:
I don't think so but there are too many helpers already